tests/Jobs/MediawikiInitTest.php

```diff
@@ -76,16 +76,17 @@ public function testSuccess()
     public function testFatalErrorIsHandled()
     {
         $mockResponse = 'oh no';
-
         $request = $this->getMockRequest($mockResponse);
 
+        $mockExceptionMessage = 'wbstackInit call for some.domain.com. No wbstackInit key in response: ' . $mockResponse;
```

Comment:
Nit: this is not a mock, but an expectation. Maybe `expectedExceptionMessage` would be more fitting?

Reply:
yes! agreed